static/js/components/widgets/HierarchicalSelectField.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from "react"
+import { sortedUniqBy, sortBy, times } from "lodash"
+
+import SelectField from "./SelectField"
+
+type Level = { label: string; name: string }
+type Option = { label: string; value: string }
```

Comment:
I think there's an `Option` interface exported from the `SelectField` module - maybe we should re-use that here?